### DISCUSSION TITLE
uefi

### DISCUSSION THREAD

**Discussion by 1392273211** [cobbler/cobbler#3667] [Q&A]
Does cobbler2.8.5 support the server to start network pxe in uefi form to install the operating system?

**Comment by SchoolGuy** [comment]
If the GRUB you are using is able to support UEFI boot then this shouldn't be an issue. Secure Boot (which most people imply) is also in modern versions of Cobbler an issue though.

**Reply by 1392273211** [reply]
Boss, does the community have a solution?

**Reply by SchoolGuy** [reply]
The solution is to just have a functioning GRUB. This is nothing that is in the scope of Cobbler. Newer versions of Cobbler are able to automatically build a version of GRUB that is network-bootable if it is installed by an OS vendor. As such I am having a hard time understanding what the issue is.

**Reply by 1392273211** [reply]
In other words, version 2.8.5 does not support uefi booting, right? So the inability to install centos8 and 9 is also because the versions are older, right? I have installed the latest version 3.2 and found that the centos7 series cannot be installed. Is there a way to solve it?

**Reply by SchoolGuy** [reply]
Cobbler 2.8.5 is too ancient for me to properly care for it. It is Python 2 based and I never was the maintainer for that major version. It may work or may not work. I can't tell for certain.

Cobbler 3.2.x should work. If you have issues installing CentOS 7 then we can attempt to fix them as Cobbler 3.2.x is a release series that I do care about and that I am partly maintaining still.

**Reply by 1392273211** [reply]
Thanks for the answer
